app/assets/javascripts/discourse/models/user.js.es6

```diff
@@ -19,6 +19,8 @@ import PreloadStore from "preload-store";
 import { defaultHomepage } from "discourse/lib/utilities";
 import { userPath } from "discourse/lib/url";
 
+export const SECOND_FACTOR_METHODS = { totp: 1, backup_code: 2 };
```

Comment:
Constants should always be upper case. I see the `SECOND_FACTOR_METHODS` is, but to be clear the properties should be too. `SECOND_FACTOR_METHODS.TOTP` for example.